src/index.d.ts

```diff
@@ -544,6 +544,48 @@ declare namespace admin.auth {
     toJSON(): Object;
   }
 
+  /**
+   * Interface representing the common properties of a user enrolled second factor.
+   */
+  interface MultiFactorInfo {
+  
+    /**
+     * The user's unique ID of the enrolled second factor.
+     */
+    uid: string;
+
+    /**
+     * The optional display name for an enrolled second factor.
```

Comment:
The optional display name of the enrolled second factor.

Reply:
Done.